.github/workflows/iroha2-dev-pr.yml

```diff
@@ -66,7 +69,10 @@ jobs:
           name: lcov.info
           path: lcov.info
 
-  integration:
+  # migrating to pytest
+  # include: client/tests/integration/
+  # exclude: client/tests/integration/extra_functional
+  client_integration:
```

Comment:
I would keep the old name. They are integration tests for all of Iroha, we may move them outside of `client` crate and pull to the root of the project

Reply:
Renaming is ok, and what do you think about which step p2p/tests/integration should be included in?

Reply:
Ah, I meant renaming from `client_integration` to `integration` is ok. Keeping the old name is fine

Reply:
> what do you think about which step should p2p/tests/integration be included in?

it could be argued both ways. Maybe you could open a separate issue for this?

Reply:
#4584